### PR TITLE
Added instructions on which file to edit.

### PR DESCRIPTION
**Pull request by kdvlr**
##### Summary
Added one line to the Telegram documentation to make it clear which file to edit. 

##### Component Name
Telegram

### CODE REVIEW

File: health/notifications/telegram/README.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ You need to:
 2.  Start a conversation with your bot or invite it into a group where you want it to send messages.
 3.  Find the chat ID for every chat you want to send messages to. Contact the [@myidbot](https://t.me/myidbot) bot and send the `/getid` command to get your personal chat ID or invite it into a group and use the `/getgroupid` command to get the group chat ID. Group IDs start with a hyphen, supergroup IDs start with `-100`.  
     Alternatively, you can get the chat ID directly from the bot API. Send *your* bot a command in the chat you want to use, then check `https://api.telegram.org/bot{YourBotToken}/getUpdates`, eg.  `https://api.telegram.org/bot111122223:7OpFlFFRzRBbrUUmIjj5HF9Ox2pYJZy5/getUpdates`
-
+4.  Set the bot token and the chat ID of the recipient in /etc/netdata/health_alarm_notify.conf (to edit it on your system run /etc/netdata/edit-config health_alarm_notify.conf), like this:
```
Comment (thiagoftsm) [inline health/notifications/telegram/README.md:22]: Hello @kdvlr ,

Please, can you add add ``  around the file to call reader attention for the filename?

Best regards!

Comment (thiagoftsm) [reply health/notifications/telegram/README.md:22]: For example, `/etc/netdata/health_alarm_notify.conf`

Comment (kdvlr) [reply health/notifications/telegram/README.md]: Thanks. Done! Please review.

**Review by thiagoftsm — APPROVED** [review]
The technical description is correct, thank you @kdvlr !

**Review by joelhans — APPROVED** [review]
Thank you very much, @kdvlr!

**Comment by OdysLam** [comment]
Hey @kdvlr, 

Thank you for this contribution and welcome to our community :) Make sure to check out our community forum at https://community.netdata.cloud !

**Review by alivx — APPROVED** [review]

**Pull request merged**